display.go

```diff
@@ -1,12 +1,24 @@
 package kvm
 
 import (
+	"errors"
 	"fmt"
 	"log"
+	"os"
+	"strconv"
 	"time"
 )
 
 var currentScreen = "ui_Boot_Screen"
+var backlightState = 0 // 0 - NORMAL, 1 - DIMMED, 2 - OFF
+
+var dim_ticker *time.Ticker
+var off_ticker *time.Ticker
+
+const (
+	TOUCHSCREEN_DEVICE      string = "/dev/input/event1"
+	BACKLIGHT_CONTROL_CLASS string = "/sys/class/backlight/backlight/brightness"
```

Comment:
Effective Go and the Go stdlib usually prefers lowerCamelCase/UpperCamelCase (depending on whether exported) for constant names. E.g., per https://go.dev/wiki/CodeReviewComments:
> See https://go.dev/doc/effective_go#mixed-caps. This applies even when it breaks conventions in other languages. For example an unexported constant is maxLength not MaxLength or MAX_LENGTH.

Reply:
Thanks! Updated.